a2lfile/src/loader.rs

```diff
@@ -1,8 +1,52 @@
+use std::ffi::OsString;
 use crate::A2lError;
 use std::fs::File;
 use std::io::Read;
 use std::path::Path;
 
+// is_numchar()
```

Comment:
None of these functions should be `pub`. They should be `pub(crate)` to make sure they do not appear in the externally visible API of a2lfile. Even though the modules `tokenizer`, `loader`, etc. are not `pub mod` it is best to be clear about the intent.

Putting these helper functions in the `loader` module is also not really appropriate, since they have nothing to do with loading. Only `make_include_filename` makes sense there.

Their original location in the tokenizer is fine, they would just have to be pub(crate) from that location.

Reply:
i reverted the _u8 renaming and i have cast the char to u8 in the a2ml tokenizer (i am not sure about the CPU processing of the casting in rust though);